### PR TITLE
Adds generic resource editing page and route

### PR DESCRIPTION
**Pull request by bipuladh**
Adds Edit button to Kebab menu

Screenshot from 2022-04-25 14-47-03
Screenshot from 2022-04-25 14-44-56

### CODE REVIEW

File: packages/odf/components/ResourceEditor/GenericResourceEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
```
Comment (GowthamShanmugam) [inline packages/odf/components/ResourceEditor/GenericResourceEditor.tsx:1]: This component is common for all, move this to share

Comment (bipuladh) [reply packages/odf/components/ResourceEditor/GenericResourceEditor.tsx:1]: It is used at a specific Route so I don't want to put it there.

Comment (GowthamShanmugam) [reply packages/odf/components/ResourceEditor/GenericResourceEditor.tsx:1]: ok

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**